File: server/src/main/java/com/chenkh/vchat/server/dao/MsgDAO.java
package com.chenkh.vchat.server.dao;

import com.chenkh.vchat.base.msg.ContenMsg;

import java.sql.Timestamp;
import java.util.List;



public interface MsgDAO  {
	
	public List<ContenMsg> getOfflineMsg(int id);
	
	
	public void addOfflineMsg(int fromId,int toId,String msg,Timestamp date);
	
	

}

File: server/src/main/java/com/chenkh/vchat/server/dao/MsgDAO.kt
package com.chenkh.vchat.server.dao

import com.chenkh.vchat.base.msg.ContenMsg
import java.sql.Timestamp

interface MsgDAO {
    fun getOfflineMsg(id: Int): List<ContenMsg>
    fun addOfflineMsg(fromId: Int, toId: Int, msg: String?, date: Timestamp?)
}